docusaurus/docs-api/ziti-sdk-swift.mdx

```diff
@@ -4,5 +4,5 @@ hide_title: true
 ------
 
 <head>
-    <meta http-equiv="refresh" content="0; URL=/docusaurus/swift/" />
```

Comment:
i feel like there are other places that this exists? did you scan the all the files?

Reply:
Now I have! :+1: